service/evidence/evidence_store_test.go

```diff
@@ -54,13 +54,11 @@ func TestStoreEvidence(t *testing.T) {
 			args: args{
 				in0: context.TODO(),
 				evidence: &evidence.Evidence{
-					Id:                "MockEvidenceId",
-					ServiceId:         "MockServiceId",
-					ResourceId:        "MockResourceId",
-					Timestamp:         timestamppb.Now(),
-					ApplicableMetrics: []int32{1, 2},
-					Raw:               "",
-					Resource:          nil,
+					Id:        "MockEvidenceId",
+					ServiceId: "MockServiceId",
+					Timestamp: timestamppb.Now(),
+					Raw:       "",
+					Resource:  nil,
```

Comment:
Do we allow the storage of evidences that do not contain a resource description?